Support array of objects for `script:ld+json` meta descriptor (#15082)
remix-run/react-router: packages/react-router/.changes/patch.support-array-script-ld-json.md

Support arrays of objects for the `script:ld+json` meta descriptor

> - Widen `"script:ld+json"` in `MetaDescriptor` from `LdJsonObject` to `LdJsonObject | LdJsonObject[]` so a route can declare multiple JSON-LD schemas (e.g. `Organization` + `BreadcrumbList`) in a single `<script type="application/ld+json">` tag emitted by `<Meta />`.

Reviewer: ```suggestion
Widen `MetaDescriptor` `script:ld+json` type from `LdJsonObject` to `LdJsonObject | LdJsonObject[]` to permit multiple JSON-LD schemas in a single `<script type="application/ld+json">` tag emitted by `<Meta />`
```
